File: coin.rb
random = rand(1..100).to_i
case random
when random == 1
  puts "penny"
when random == 5
  puts "nickel"
when random == 10
  puts "dime"
when random == 25
  puts "quarter"
else
  puts "no coin"
  puts random
end

File: coin.swift
//1-100
import Foundation

var x: Int? = Int(arc4random_uniform(100) + 1)
if x == 1 {
  print("penny")
}
else if x == 5 {
  print("nickel")
}
else if x == 10 {
  print("dime")
}
else if x == 25 {
  print("quarter")
}
else {
  print("no coin")
  print(x!)
}
